cpp/BUILD.bazel

```diff
@@ -123,3 +125,16 @@ cc_binary(
         "@com_google_googletest//:gtest_main",
     ],
 )
+
+cc_test(
+    name = "ray_register_test",
+    srcs = glob([
+        "src/ray/test/ray_register_test.cc",
+    ]),
+    copts = COPTS,
+    linkstatic = False,
+    deps = [
+        "ray_api",
+        "@com_google_googletest//:gtest_main",
+    ],
+)
```

Comment:
Better to add this test to an independent folder? Otherwise we need to change this bazel file every time when we add new test files.